lib/features/feedback/ui/feedback_screen.dart

```diff
@@ -0,0 +1,179 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fluttercon/common/repository/api_repository.dart';
+import 'package:fluttercon/common/utils/constants/app_assets.dart';
+import 'package:fluttercon/core/theme/theme_colors.dart';
+import 'package:fluttercon/features/feedback/cubit/send_feedback_cubit.dart';
+import 'package:fluttercon/features/feedback/widgets/emoji_container.dart';
+import 'package:fluttercon/features/feedback/widgets/feedback_custom_appbar.dart';
+import 'package:go_router/go_router.dart';
+
+class FeedbackScreen extends StatefulWidget {
+  const FeedbackScreen({super.key});
+
+  @override
+  State<FeedbackScreen> createState() => _FeedbackScreenState();
+}
+
+class _FeedbackScreenState extends State<FeedbackScreen> {
+  final TextEditingController feedbackController = TextEditingController();
+  int? selectedRating;
+  int? selectedEmojiIndex;
+
+  @override
+  void dispose() {
+    // Dispose of the controller to avoid memory leaks
+    feedbackController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
```

Comment:
Register this cubit in the `providers` array in `lib/bootstrap.dart` and remove this `BlocProvider` widget here. Let it start with the `Scaffold` directly.